### ISSUE TITLE
Scene - Edit Defaults

### ISSUE THREAD

**Issue by kmavridis** [zalo/CascadeStudio#127]
Hello, 
how can i remove fog from the scene, add more light to it and make bigger the view frustum so my whole model to be visible?

Thank you in advanced

**Comment by raydeleu** [comment]
I agree that the settings of the fog are not always ideal. It depends on the size of your object, but if the size of your object becomes too large the fog will cover smaller parts. 

<img width="360" alt="fog-cs" src="https://user-images.githubusercontent.com/38007983/173207881-997b0605-b9f2-4180-86c0-3bf05e29e9a6.png">

To change this you would have to download install the code locally on your computer as described in https://github.com/raydeleu/CascadeStudioManual#22-local-installation

In https://github.com/raydeleu/CascadeStudioManual#133-modifying-the-3d-view I have described how you can change some default values of the 3D window. I guess you would need to tinker around in the code of https://github.com/zalo/CascadeStudio/blob/master/js/MainPage/CascadeView.js to see what the effect is. The following lines describe some of the behaviour of the fog in the scene. 

```` javascript

    // Expand fog distance to enclose the current object; always expand
    //  otherwise you can lose the object if it gets smaller again)
    this.boundingBox = new THREE.Box3().setFromObject(this.mainObject);
    this.fogDist = Math.max(this.fogDist, this.boundingBox.min.distanceTo(this.boundingBox.max)*1.5);
    this.environment.scene.fog = new THREE.Fog(this.environment.backgroundColor, this.fogDist, this.fogDist + 400);

```` 

Recently there was a change to the code that allows to remove the grid and the ground plane from the scene. Perhaps the author https://github.com/drb3st could add this option to the interface as well?
